serve-json/server.js

```diff
@@ -0,0 +1,39 @@
+var http = require('http')
+var fs = require('fs')
+var async = require('async')
+var path = require('path')
+
+var port = 3000
+
+var server = http.createServer(function (req, res) {
+  res.setHeader('Content-Type', 'application/json')
+
+  async.each([
+    'du.json',
+    'dum.json',
+    'dummy.json'
+  ], function (file, cb) {
+    var filePath = path.join(__dirname, 'data/' + file)
+    var content = fs.readFileSync(filePath)
```

Comment:
When this gets read, content will be a Buffer and not a JSON object

You need to JSON.parse(content.toString()) to get the JSON content